src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -816,5 +809,12 @@ unique_ptr<Scene> svgSceneBuild(SvgNode* node, float& vx, float& vy, float& vw,
     auto root = Scene::gen();
     root->push(move(compositeLayer));
 
+    loaderData.doc->node.doc.vx = vBox.x;
+    loaderData.doc->node.doc.vy = vBox.y;
+    loaderData.doc->node.doc.vw = vBox.w;
+    loaderData.doc->node.doc.vh = vBox.h;
+    loaderData.doc->node.doc.w = w;
+    loaderData.doc->node.doc.h = h;
```

Comment:
I have question. Why do we need to update the value of loaderData here?

Reply:
If there were no any given viewbox & size information, It finds the default size from the scene geometry. To get the final size from it, it updates the info at the last.

Reply:
Are there any cases where loaderData needs to be updated again?
The loader do the parsing again, so there is no need to reuse this.
Even if it is modified to be reused(in the future?), it seems that redundant calculations will occur because w, and h are calculated once again in the above logic.

Reply:
@JSUYA this is a fix for the regression of https://github.com/thorvg/thorvg/pull/1248

Reply:
@JSUYA and this one. It needs the get the completed size by this https://github.com/thorvg/thorvg/pull/1338

Reply:
I missed the logic overriding, I will update it again soon. 👍

Reply:
@JSUYA updated thanks!